src/index.html

```diff
@@ -0,0 +1,28 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title> JS-Capstone </title>
+</head>
+<body id="body">
+    <nav class="nav flexbox">
+        <a href=""><img src="./assets/show-time.png" alt="TV logo" id="logo"></a>
+        <div class="menu flexbox">
+            <a href="" id="item-count">Movies</a>
+            <a href="">Series</a>
+            <a href="">Anime</a>
+        </div>
+    </nav>
```

Comment:
- [ ] Kindly wrap this `nav` into a `header` tag to follow the HTML semantic tags due to SEO purpose and HTML best practices :blush:

Reply:
Thank you very much dear reviewer